### PR TITLE
BreadcrumbItem Component

### PR DESCRIPTION
**Pull request by nataliepina**
The `BreadcrumbItem` component can be utilized to reduce common formatting utilizing FlexItems, frequently used within Kommander UI `Breadcrumb` component usage. 

`BreadcrumbItem` offers the option of an icon, url, onClick functionality, and children. All with formatted style to align with common usage. 

I have reduced the Stories to simply display the previous without and with `BreadcrumbItem`, for the sake of reducing our story files any additional examples are probably superfluous here. 

Changes to `main.js` are to ensure stories display their code examples in the Docs tab. 

I have also corrected some spelling/grammar errors in the contributing documentation while reading through for info on adding a new component. 

Closes D2IQ-65928

## Testing

Run Storybook and view BreadcrumbItem story. 

## Checklist for PR creator

- [x] If any new components were added, there are exported from `packages/index.ts`
- [x] If this PR is associated with a JIRA, it is mentioned in commit message footer ("Closes …")
- [ ] If this PR contains breaking changes, is stated in commit message body ("BREAKING CHANGE: …")
- [x] Info for applicable sections above is provided

### CODE REVIEW

**Review by vacas5 — APPROVED** [review]
nice!

**Review by mstruebing — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -83,13 +83,13 @@ When a new component is being created, work with the design team to document des
 #### Consider the Following Before Adding a New Component
 
 **Is it likely this component be used in more than 1 app?**
-For example, a component for a card that may contain any content would be a good candidate for ui-kit. A card that specifically renders data specific to an object in an napp is not.
+For example, a component for a card that may contain any content would be a good candidate for ui-kit. A card that specifically renders data specific to an object in an app is not.
```
Comment (clintonmedbery) [inline CONTRIBUTING.md:86]: I could have used you in college 😂

**Review by clintonmedbery — APPROVED** [review]
Nice work!

File: CONTRIBUTING.md (hunk 1 above)
Comment (nataliepina) [reply CONTRIBUTING.md:86]: I took a copy editing course with my Journalism minor that will forever haunt me. 😆

**Pull request merged**